programmers/K번째수.js

```diff
@@ -0,0 +1,7 @@
+function solution(array, commands) {
+  let answer = commands.map((val) => {
+    let [i, j, k] = val;
```

Comment:
아니.. 아주 잘하시네요 준상님 ㅋㅋㅋㅋ??
저보다 잘하시는 듯!!
변수 선언은 무조건 const로 한다고 생각해주세요.
중간에 변경되어야할 때만 let으로 선언한답니다 :D

타 언어랑은 쬐끔 다르죠?ㅋㅋㅋ 이게 자스 컨벤션입니다 ^^
```suggestion
  const answer = commands.map((val) => {
    const [i, j, k] = val;
```

Reply:
변수를 처음부터 const로 만들고 변수가 변할 수 있을때 마다 let으로 바꿔서 써야하는건지 이제 알았습니다. 아하